test/00.hello_world/test_serialize_driver.py

```diff
@@ -0,0 +1,47 @@
+import freetensor as ft
+import pytest
+from random import randint
+
+
+def test_basic():
+    l, r = -2147483648, 2147483647
+    for i in range(10000):
+        # CPU
+        target = ft.CPU(randint(0, 1))
+        txt = ft.dump_target(target)
+        print(txt)
+        target2 = ft.load_target(txt)
+        assert target == target2
```

Comment:
It's a better practice to separate different assertions into different tests with clear names. If one assertion fails, one can easily tell what is wrong.